src/drivers/intel/ssp/ssp.c

```diff
@@ -1000,8 +1011,19 @@ static void ssp_stop(struct dai *dai, int direction)
 
 	spin_lock(&dai->lock);
 
-	/* wait to get valid fifo status */
-	wait_delay(PLATFORM_SSP_DELAY);
+	/*
+	 * Wait to get valid fifo status in clock consumer mode. TODO it's
+	 * uncertain which SSP clock consumer modes need the delay atm, but
+	 * these can be added here when confirmed.
+	 */
+	switch (ssp->config.format & SOF_DAI_FMT_CLOCK_PROVIDER_MASK) {
+	case SOF_DAI_FMT_CBC_CFC:
+		break;
+	default:
+		/* delay for all SSP consumed clocks atm - see above */
```

Comment:
only minor . typo consumed -> consumer

Reply:
good change @lgirdwood we should add this patch as a separate PR, it might help even outside of multi-core changes.

Reply:
What is the motivation of the change here? the PLATFORM_SSP_DELAY delay is only some us, no?

Reply:
The delay is NOT needed in SSP clock producer mode. It can be 100uS.